subsys/net/ip/Kconfig.ipv6

```diff
@@ -156,6 +156,14 @@ config NET_IPV6_DAD
 	  The value depends on your network needs. DAD should normally
 	  be active.
 
+config NET_IPV6_RS_TIMEOUT
+	int "Timeout of Router Solicitation messaging"
+	depends on NET_IPV6_ND
+	default 1
```

Comment:
Should we put here allowed range, so something like this
```
range 1 $(UINT32_MAX)
```
The upper value is way too large, but it is difficult to say what should be the max value.

Reply:
The time is in seconds, so it feels like a maximum limit on 5-10 second would suffice? Else something else is broken.

Reply:
OK, I have now added `range 1 30`.